vue/components/ui/molecules/tabs/tabs.vue

```diff
@@ -152,6 +152,7 @@ export const Tabs = {
         tabClasses(tab) {
             const base = { tab: true };
             base[`tab-${tab.value}`] = true;
+            base[`tab-${tab.value.replace(" ", "-").toLowerCase()}`] = true;

```

Comment:
Use https://github.com/ripe-tech/ripe-components-vue/blob/master/js/util.js#L1

Reply:
I made this change so that we have improved classes, normalize wouldn't solve this example:
<img width="621" alt="imagem" src="https://user-images.githubusercontent.com/22588915/94898752-fdcc7e00-0489-11eb-8a8f-861b3ca8e1f4.png">
would become:
<img width="621" alt="imagem" src="https://user-images.githubusercontent.com/22588915/94898837-1dfc3d00-048a-11eb-8acb-479d3215dcd0.png">

Taking a better look, we should probably do something like this in for section classes too and use something that handles invalid characters too

Reply:
Move this to a `buildSlug` function inside `util`

Reply:
Done, for example, "Column #1 Section #1" now parses to "column-1-section-1"